packages/web3-eth/test/integration/get_revert_reason.test.ts

```diff
@@ -96,9 +96,9 @@ describe('Web3Eth.getRevertReason', () => {
 					data: '000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000155468697320697320612073656e64207265766572740000000000000000000000',
 				});
 				break;
-			case 'ganache':
+			case 'hardhat':
 				expect(response).toMatchObject({
-					reason: 'VM Exception while processing transaction: revert This is a send revert',
+					reason: "Error: VM Exception while processing transaction: reverted with reason string 'This is a send revert'",
```

Comment:
this error message change is due to our lib?

Reply:
no hardhat outputs a slightly different error message